average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+   const newArr = numbers.filter( value => !Number.isNaN(value) );
+  return newArr.reduce((p, c)=> p + c, 0) / newArr.length;
 }
 
 module.exports = {average};
```

Comment:
Try a new test: The tests in `test/average.test.js` express the functionality of the `average` function.
Consider this situation: In IoT applications, analog sensors will report occasional outliers. 'Outliers' are values so small or so large, they aren't expected to happen. We need new functionality to ignore these outliers.
First add a unit-test in `test/average.test.js` to supply outliers and expect them to be ignored. Name the test appropriately. Then implement it in the code.
You'll need thresholds to recognize outliers: Think of sensing battery-temperature values and determine reasonable values.